src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/generated.py

```diff
@@ -54,11 +54,11 @@
 cli_command('webapp deployment user show', ProviderOperations.get_publishing_user, factory)
 
 factory = lambda _: web_client_factory().server_farms
-cli_command('webapp plan create', create_app_service_plan)
-cli_command('webapp plan update', update_app_service_plan)
-cli_command('webapp plan delete', ServerFarmsOperations.delete_server_farm, factory)
-cli_command('webapp plan list', ServerFarmsOperations.get_server_farms, factory)
-cli_command('webapp plan show', ServerFarmsOperations.get_server_farm, factory)
+cli_command('app-service-plan create', create_app_service_plan)
```

Comment:
I think the logic is that a webapp belongs to a service-plan but with the old structure it looked like service-plan was a child of webapp.  It might be we have "az webapp service-plan" and "az webapp site" or something. I would agree that having "peers" at the top-level probably clutters our top-level structure.

---
In reply to: [79496925](https://github.com/Azure/azure-cli/pull/924#discussion_r79496925) [](ancestors = 79496925)